### PR TITLE
Adds 'Explorer' Job [WIP]

### PR DESCRIPTION
**Pull request by MisterLayne**
Adds 'Explorer' job to game.
Pushed it for community feedback.

To Do:
- [ ] Add area for Explorer to spawn
- [ ] Add new Z-Levels for explorer to explore (maybe)
- [ ] Add some more customized loadout options for explorer

### CODE REVIEW

**Comment by Anewbe** [comment]
Honest Question: Why?

**Comment by mistyLuminescence** [comment]
👍

**Comment by MisterLayne** [comment]
> Honest Question: Why?
It's like Xenoarcheology without the mining. Hiding interesting stuff in space for people to find, use for research, and study. Creates interesting scenarios and If you're going to be stuck on an asteroid in space, might as well be able to explore artifacts and history around you.

**Comment by Anewbe** [comment]
You are aware that we're moving away from the asteroid map, yes?

**Comment by Belsima** [comment]
Maybe have disabled like Geneticist for now.

**Comment by mistyLuminescence** [comment]
Honestly, this seems like it'd be even better for Southern Cross, there's so much to explore planetside there.

**Comment by ElectricB1ue** [comment]
Yeah, this seems like a SC job if anything. I'd probably play it

**Comment by MisterLayne** [comment]
I was hoping to keep it until Southern Cross came out, honestly. Still hoping it could be useful in the last month or two of Polaris.

> Why is space invader an alt title

@mistyLuminescence

**Comment by mistyLuminescence** [comment]
I WAS JOKING

File: code/game/jobs/job/science.dm [hunk 1]
```diff
@@ -86,3 +86,23 @@
...
+	selection_color = "#9b5f9b"
+	idtype = /obj/item/weapon/card/id/science/roboticist
+	economic_modifier = 5
+	access = list(access_eva, access_external_airlocks, access_teleporter, access_research, access_tech_storage, access_tcomsat, access_xenoarch)
+	minimal_access = list(access_eva, access_external_airlocks, access_teleporter, access_research, access_tech_storage, access_tcomsat, access_xenoarch)
+	minimum_character_age = 20
+	minimal_player_age = 12
+
+	outfit_type = /decl/hierarchy/outfit/job/science/explorer
+	alt_titles = list("Pioneer", "Space Invader", "Expeditioner")
```
Comment (MisterLayne) [inline code/game/jobs/job/science.dm]: Note to self: Remove "Space Invader".

**Comment by Anewbe** [comment]
I have other ways I'd rather do this, when we move to the Southern Cross.

**Comment by Anewbe** [comment]
Alright. We will not be merging this as-is. We do like the idea of an "Explorer" role, but not yet, and not quite like this. @MisterLayne if you want, we'll let you know what we want done when we're ready for the role, otherwise I can probably do it myself.

I will be closing this by the end of tomorrow, unless you choose to do so yourself.

**Pull request closed without merging**